test/extended/operators/olm.go

```diff
@@ -230,4 +231,46 @@ var _ = g.Describe("[sig-operator] an end user use OLM", func() {
 		}
 
 	})
+
+	// OCP-21126 - Subscription status says 'CSV is installed' when it is not
+	// author: scolange@redhat.com
+	g.It("Subscription status says CSV is installed", func() {
```

Comment:
@scolange Could you help update it by following the title naming rule? Thanks! 
Team-Importance-TestCaseID-TestCaseTitle, split with ‘-’